docs/api/field_type_api_and_best_practices.md

```diff
@@ -613,21 +613,22 @@ Here is an example for **ezurl** field type:
 ``` yaml
 parameters:
     ezpublish.fieldType.ezurl.externalStorage.class: eZ\Publish\Core\FieldType\Url\UrlStorage
-
 services:
     ezpublish.fieldType.ezurl.externalStorage:
         class: %ezpublish.fieldType.ezurl.externalStorage.class%
+        arguments:
+            - "@ezpublish.fieldType.ezurl.storage_gateway"
         tags:
             - {name: ezpublish.fieldType.externalStorageHandler, alias: ezurl}
 ```
 
-The configuration is straight forward. Nothing specific except the **`ezpublish.fieldType.externalStorageHandler `** tag, the `alias` attribute still begin the *fieldTypeIdentifier*.
+The configuration requires providing the **`ezpublish.fieldType.externalStorageHandler `** tag, with the `alias` attribute being the *fieldTypeIdentifier*. You also have to inject the gateway in the `arguments` tag, [see below](#gateway-based-storage_1).
```

Comment:
I wouldn't call `arguments` key a "tag". Tag is a keyword in Symfony DI Container domain. Either "pass a gateway service definition as an argument" or just remove word "tag" (to me, sounds good w/o it as well).